cmd/lotus-storage-miner/info.go

```diff
@@ -158,6 +179,30 @@ func infoCmdAct(cctx *cli.Context) error {
 		}
 	}
 
+	if cctx.Bool("show-rewards") {
+		curTs := head
+		var blockRewardsEarned = abi.NewTokenAmount(0)
+		for curTs.Height() > abi.ChainEpoch(0) {
+			stout, err := api.StateCompute(ctx, curTs.Height(), nil, curTs.Key())
```

Comment:
I'm pretty sure this will take ~1 week on mainnet

Reply:
lol😂 is there any other way to do this? Can we add a map in reward actor?